common/src/main/java/com/wynntils/features/user/redirects/HorseRedirectFeature.java

```diff
@@ -0,0 +1,51 @@
+/*
+ * Copyright © Wynntils 2022.
+ * This file is released under AGPLv3. See LICENSE for full license details.
+ */
+package com.wynntils.features.user.redirects;
+
+import com.wynntils.core.features.UserFeature;
+import com.wynntils.core.features.properties.FeatureCategory;
+import com.wynntils.core.features.properties.FeatureInfo;
+import com.wynntils.core.notifications.NotificationManager;
+import com.wynntils.mc.utils.ComponentUtils;
+import com.wynntils.wynn.event.ChatMessageReceivedEvent;
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+import net.minecraft.ChatFormatting;
+import net.minecraft.network.chat.TranslatableComponent;
+import net.minecraftforge.eventbus.api.EventPriority;
+import net.minecraftforge.eventbus.api.SubscribeEvent;
+
+@FeatureInfo(category = FeatureCategory.REDIRECTS)
+public class HorseRedirectFeature extends UserFeature {
+    // Strings and patterns that we get from Wynncraft.
+    private static final String noRoomString = "There is no room for a horse.";
+    private static final String despawnedString = "Since you interacted with your inventory, your horse has despawned.";
+    private static final Pattern NO_ROOM_PATTERN = Pattern.compile(noRoomString);
+    private static final Pattern HORSE_DESPAWNED_PATTERN = Pattern.compile(despawnedString);
```

Comment:
Just a reminder, no need to separate the strings and patterns here. 

Use colored messages in the patterns to make sure it works.